source/Ship.cpp

```diff
@@ -4401,9 +4401,19 @@ double Ship::CalculateDeterrence() const
 			const Outfit *weapon = hardpoint.GetOutfit();
 			if(weapon->Ammo() && weapon->AmmoUsage() && !OutfitCount(weapon->Ammo()))
 				continue;
-			double strength = weapon->ShieldDamage() + weapon->HullDamage()
-				+ (weapon->RelativeShieldDamage() * attributes.Get("shields"))
-				+ (weapon->RelativeHullDamage() * attributes.Get("hull"));
+			// Ignore disabled and asteroid damages because they don't apply here.
+			// Ignore fuel and slowing damage because they don't affect combat ability.
```

Comment:
That's just not true?
If the ship in question relies on Flamethrowers, or other fuel consuming weapons, fuel damage will absolutely affect its combat ability.
For slowing damage, having a significantly reduced turn rate will definitely affect the combat ability of a ship that relies on fixed guns, especially if the projectiles do not have good homing or manoeuvrability.
Also, no mention of leak?

Reply:
~~as if we're going to take all of that into account~~
we could at some point but this is already a big improvement, although let me rephrase that
```suggestion
			// Ignore fuel, leak and slowing damage for now, as their affect on combat abilities varies.
```
I mean we consider weapons that dont have gas to fire for scaring pirates anyway - how would I measure all of this stuff tho, feels too complex for this PR